types/src/proof/position/position_test.rs

```diff
@@ -259,27 +259,27 @@ fn slow_get_frozen_subtree_roots_impl(root: Position, max_leaf_index: u64) -> Ve
     }
 }
 
-fn slow_get_frozen_subtree_roots(num_leaves: usize) -> Vec<Position> {
+fn slow_get_frozen_subtree_roots(num_leaves: u64) -> Vec<Position> {
     if num_leaves == 0 {
         Vec::new()
     } else {
-        let max_leaf_index = num_leaves as u64 - 1;
+        let max_leaf_index = num_leaves - 1;
         let root = Position::root_from_leaf_count(num_leaves);
         slow_get_frozen_subtree_roots_impl(root, max_leaf_index)
     }
 }
 
 #[test]
 fn test_frozen_subtree_iterator() {
-    for n in 0..10000usize {
+    for n in 0..10000 {
         assert_eq!(
             FrozenSubTreeIterator::new(n).collect::<Vec<_>>(),
             slow_get_frozen_subtree_roots(n),
         );
     }
 }
 
-fn collect_all_positions(num_leaves: usize, num_new_leaves: usize) -> Vec<u64> {
+fn collect_all_positions(num_leaves: u64, num_new_leaves: u64) -> Vec<u64> {
```

Comment:
why not `LeafCount`?

@bors-libra delegate+

Reply:
I didn't realize it was in scope, but it is so ... done.